test/tc_run_pod_test.go

```diff
@@ -76,12 +76,4 @@ func (e *e2e) testCaseRunPod([]string) {
 
 	e.logf("Waiting for test pod to be ready")
 	e.waitFor("condition=ready", "pod", "--all")
-
```

Comment:
I wonder if the test case as a whole has much of value right now?
If we want to keep a negative test around maybe we could add the failure case:
```
	failureOutput := e.kubectlFailure(
		"exec", examplePodName, "--", "rmdir", "/home",
	)
	e.Contains(failureOutput,
		"rmdir: failed to remove '/home': Operation not permitted",
	)
```
to `testCaseBaseProfile` ?

Reply:
Yeah I agree and decided to remove the whole test case for now.